test/tests_for_auth/ee2_admin_mode_test.py

```diff
@@ -128,7 +121,11 @@ def get_user_mocks(
     def get_client_mocks(self, *to_be_mocked):
         return _get_client_mocks(self.cfg, self.config_file, *to_be_mocked)
 
-    @patch.object(Catalog, "get_module_version", return_value="module.version")
```

Comment:
These patches could never have worked since `get_module_version` returns `dict`, not `str`. They became active once I started using the getter for the new catalog client, so I fixed them and removed the previously working patch in the `setUp` method. This way is less confusing since the patch is right above the method signature rather than in the `setUp` method and only applies to the relevant tests instead of the entire module (and if I understand correctly, any later tests if `tearDown` fails or isn't run for some reason.

Reply:
> and if I understand correctly, any later tests if tearDown fails or isn't run for some reason.

mildly worrying!